README.md

```diff
@@ -206,17 +167,3 @@ Check out our [coding guidelines](https://lion-web.netlify.app/guides/principles
 
 Lion Web Components are only as good as its contributions.
 Read our [contribution guide](https://github.com/ing-bank/lion/blob/master/CONTRIBUTING.md) and feel free to enhance/improve Lion. We keep feature requests closed while we're not working on them.
-
-## Contact
-
-Feel free to create a github issue for any feedback or questions you might have.
```

Comment:
Maybe keep mention of "Feel free to create a github issue for any feedback or questions you might have." ?